docs/components/User/index.rst

```diff
@@ -9,3 +9,8 @@ Users management implementation in PHP.
    installation
    models
    basic_usage
+
+Learn more
+----------
+
+* :doc:`Customers & Users in Sylius platform </book/customers/index>` - concept documentation
```

Comment:
In `docs/index.rst` you wrote `[...] in **the** Sylius platform` which is better IMO.